### DISCUSSION TITLE
Getting errored out when trying  add a External decision.

### DISCUSSION THREAD

**Discussion by RukmalS92** [gorules/zen#215] [Q&A]
Getting errored out when trying add a external decision inside deicision. how can I resolve this?
 
<img width="1366" alt="Screenshot 2024-07-17 at 9 10 19 AM" src="https://github.com/user-attachments/assets/dd2ba305-4c1b-410e-af2f-21e73b95e7a4">

**Comment by ivanmiletic** [comment]
@RukmalS92 please publish external decision first - we will enhance UI further for using external decisions
